tensorflow/contrib/model_average/__init__.py

```diff
@@ -0,0 +1,19 @@
+# 2017 Contrib..
+# ==============================================================================
+"""A module containing model average optimization routines."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+# pylint: disable=wildcard-import
```

Comment:
I don't think disabling wildcard is necessary here? Just explicitly import the two symbols.